docs/content/dev/config.md

```diff
@@ -0,0 +1,76 @@
+# Config
+
+The configuration of HedgeDoc is handled entirely by environment variables. Most of these variables are prefixed with `HD_` (for HedgeDoc).
+
+## Configuring HedgeDoc for development
+
+NestJS - the framework we use - is reading the variables from the environment and also from the `.env` file in the root of the project. To make it easy to get the project started we included an example `.env` file in the root of this project called `.env.example`.#
+
+If you need something else for your development endeavour please refer to our [user documentation](../configuration.md) for help in how to configure HedgeDoc appropriately.
```

Comment:
I'm not sure if this is appropriate here. The user docs currently still reflect the old 1.x config.

Reply:
Okay so just delete this sentence?